### PR TITLE
feature: adding support for new relic and lighstep

### PR DESCRIPTION
**Pull request by xoscar**
This PR adds support for both new relic and light step to show up as data store options under the settings page.

## Changes

- Updates the UI to show a different input for the list
- Adds new relic and lightstep

## Fixes

- #1793 

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

https://www.loom.com/share/4c289d7351f846029d2521958029a15b

### CODE REVIEW

File: server/model/data_stores.go [hunk 1]
```diff
@@ -77,6 +79,8 @@ func DataStoreFromConfig(dsc config.TracingBackendDataStoreConfig) DataStore {
 		ds.Type = openapi.OPEN_SEARCH
 	case signalfx:
 		ds.Type = openapi.SIGNAL_FX
+	case newrelic:
+	case lighstep:
```
Comment (jorgeepc) [inline server/model/data_stores.go:83]: If you have `newrelic` or `lighstep` in your config file, is this going to change the type to `otlp`? Do we want that behavior?

**Review by jorgeepc — COMMENTED** [review]
Looks great. I added one question and a minor suggestion: I would put `OpenSearch` and `Elastic` together in the list.

**Comment by xoscar** [comment]
> Looks great. I added one question and a minor suggestion: I would put `OpenSearch` and `Elastic` together in the list.

I added them based on the list from the designs, I'll put them together 😄

File: server/model/data_stores.go (hunk 1 above)
Comment (xoscar) [reply server/model/data_stores.go]: Nice catch, fixed!

**Review by jorgeepc — APPROVED** [review]

**Pull request merged**